### PR TITLE
Cra config from file

### PR DESCRIPTION
**Pull request by daniel-butler-irl**
### Description

Allows passing the CRA config from a file. Also added support for using multiple target configurations when using the file. 

#### Changes that affect the core Terraform module or submodules
- [ ] Bug fix
- [x] New feature
- [ ] Dependency update

#### Changes that don't affect the core Terraform module or submodules
- [ ] Examples or tests (addition or updates of examples or tests)
- [ ] Documentation update
- [x] CI-related update (pipeline, etc.)
- [ ] Other

#### Release required?
Identify the type of release. For information about the changes in a semantic versioning release, see [Release versioning](https://terraform-ibm-modules.github.io/documentation/#/versioning).

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content

If a release is required, replace this text with information that users need to know about the release. Write the release notes to help users understand the changes, and include information about how to update from the previous version.

Your notes help the merger write the commit message for the PR that is published in the release notes for the module.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### Merge actions for mergers

- Use a relevant [conventional commit](https://www.conventionalcommits.org/) message that is based on the PR contents and any release notes provided by the PR author. The commit message determines whether a new version of the module is needed, and if so, which semver increment to use (major, minor, or patch).
- Merge by using "Squash and merge".

### CODE REVIEW

**Comment by daniel-butler-irl** [comment]
Working as expected, see sample output here: https://github.com/terraform-ibm-modules/temp-test/actions/runs/5222930547/jobs/9429149827#step:6:144

File: .github/workflows/terraform-test-pipeline.yml [hunk 1]
```diff
@@ -90,45 +94,153 @@ jobs:
...
+              done
             else
-              echo "CRA IGNORE FILE: ${CRA_IGNORE_GOALS_FILE}"
-              ci/run-cra.sh "$target"
+              echo "No input environment variables to export"
+            fi
+
+            # Install CRA plugin if not detected
+            if ! ibmcloud plugin show cra; then
+              ibmcloud plugin install cra -f
```
Comment (toddgiguere) [inline .github/workflows/terraform-test-pipeline.yml:225]: Looks like this install of the plugin will not occur if a yaml config is specified? It looks like a precautionary step, but you could move the logic to very top of script to make sure it always runs.

**Review by toddgiguere — CHANGES_REQUESTED** [review]

**Review by toddgiguere — APPROVED** [review]

**Pull request merged**